### ISSUE TITLE
1.3.0 doesn't work on Windows

### ISSUE THREAD

**Issue by Marclev78** [christophehurpeau/deep-freeze-es6#6]
Using `yarn` to pull dependency, haven't tested with `npm`.

1.2.0 worked fine, but when trying to do `yarn install` with 1.3.0 on Windows I get this:

```
error An unexpected error occurred: "https://registry.yarnpkg.com/deep-freeze-es6/-/deep-freeze-es6-1.3.0.tgz: ENOENT: no such file or directory, open 'C:\\Users\\marcl\\AppData\\Local\\Yarn\\Cache\\v1\
pm-deep-freeze-es6-1.3.0-20c35c17f394ac928bdaff7b2a171eb6c792fcec\\typings\\import-*-should-not-pass.ts'".
```

Have tried cleaning cache and issue still occurs. I'd imagine it's because of the * in `import-*-should-not-pass.ts`, which is not a legal filename on Windows.

Yarn version: 1.9.4
Node version: v8.6.0
deep-freeze-es-6: 1.3.0
Windows version: Windows 10 ver 1803 (OS Build 17134.165) - latest update basically.

Edit: It appears 1.2 doesn't work with TypeScript, so I've rolled back to 1.1 for now.

**Comment by christophehurpeau** [comment]
I see the problem, I'll make a fix soon ! Thanks for reporting

**Comment by christophehurpeau** [comment]
@Marclev78 it should be fixed with 1.3.1, can you confirm ?

**Comment by Marclev78** [comment]
Can confirm 1.3.1 works fine.

Thanks for the quick action!
